build.py

```diff
@@ -362,7 +362,9 @@ def _check_arch(filename: str) -> str | None:
         with zipfile.ZipFile(filename) as zipf:
             arch_files = []
             for name in zipf.namelist():
-                if name.endswith((".so", ".dylib")) or ".so." in name:
+                if "/tests/" in name:
```

Comment:
Is this an optimization to simply ignore any files under tests?

Reply:
ah yes should have commented on this -- delocate contains some `.dylib` test files in its distribution which (intentionally) don't match the architecture of the wheel